### PR TITLE
feat: added static Login and Signup page

### PR DESCRIPTION
**Pull request by mayuresh-tech**

### CODE REVIEW

**Review by Abhijeet43 — APPROVED** [review]

File: src/components/auth/login-box/LoginBox.jsx [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+          <input
+            id="email-valid"
+            type="text"
+            class="form-input-field"
+            placeholder="user@example.com"
+            required
+          />
+          {/* <span class="validation-errors error-msg"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512">
+                        <path
+                            d="M256 0C114.6 0 0 114.6 0 256s114.6 256 256 256s256-114.6 256-256S397.4 0 256 0zM232 152C232 138.8 242.8 128 256 128s24 10.75 24 24v128c0 13.25-10.75 24-24 24S232 293.3 232 280V152zM256 400c-17.36 0-31.44-14.08-31.44-31.44c0-17.36 14.07-31.44 31.44-31.44s31.44 14.08 31.44 31.44C287.4 385.9 273.4 400 256 400z" />
```
Comment (BALAJI141031) [inline src/components/auth/login-box/LoginBox.jsx:21]: remove commented code

File: src/components/auth/signup-box/SignupBox.css [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    align-items: center;
+    margin-top: 4.2rem;
+}
+
+.auth-box-signup {
+    background-color: var(--color-white);
+    padding: 1rem 1.5rem 2.5rem 1.5rem;
+    height: auto;
+    width: 30%;
+    box-shadow: rgba(60, 64, 67, 0.3) 0px 1px 2px 0px,
```
Comment (BALAJI141031) [inline src/components/auth/signup-box/SignupBox.css:15]: you can create variable for shadow

**Review by BALAJI141031 — APPROVED** [review]

File: src/components/auth/login-box/LoginBox.jsx (hunk 1 above)
Comment (mayuresh-tech) [reply src/components/auth/login-box/LoginBox.jsx:21]: ok

File: src/components/auth/signup-box/SignupBox.css (hunk 2 above)
Comment (mayuresh-tech) [reply src/components/auth/signup-box/SignupBox.css:15]: noted

**Pull request merged**